### PR TITLE
fixes lack of space between sentences in no notifications message

### PR DESCRIPTION
**Pull request by mrniaboc**

### CODE REVIEW

**Comment by mrniaboc** [comment]
zooniverse typo

File: app/pages/notifications/index.jsx [hunk 1]
```diff
@@ -12,7 +12,7 @@ counterpart.registerTranslations('en', {
     header: 'Notifications',
     title: 'My Notifications',
     signedOut: 'You\'re not signed in.',
-    noNotifications: 'You have no notifications.',
+    noNotifications: 'You have no notifications. ',
```
Comment (camallen) [inline app/pages/notifications/index.jsx]: Best to put the space between the translated text strings here instead.
https://github.com/zooniverse/Panoptes-Front-End/blob/b59be6cd8c20054534722dc78d221d05c33ec0c7/app/pages/notifications/index.jsx#L113

Comment (simoneduca) [reply app/pages/notifications/index.jsx]: This has broken the test, which explain why Travis is failing.

**Comment by jelliotartz** [comment]
I understand what needs to be done to fix this typo per @camallen's suggestion, and am happy to push a commit on here to get this PR moving, if it's helpful :) - I'm also happy to open a new PR if that's a better workflow.

**Comment by simoneduca** [comment]
Beat me to it @jelliotartz, nice one. I'd open a new PR, but it's up to you.

**Comment by camallen** [comment]
@jelliotartz just commit here to get it done. I'm sure grant won't worry.

**Comment by jelliotartz** [comment]
Staged [here](https://notifications-typo.pfe-preview.zooniverse.org/notifications).

**Pull request merged**